File: backend/Models/Worker.cs
using System;
using System.Collections.Generic;

namespace backend.Models;

public partial class Worker
{
    public int IdWorker { get; set; }

    public string Name { get; set; } = null!;

    public string Surname { get; set; } = null!;

    public string? Patronymic { get; set; }

    public bool IsDeleted { get; set; }

    public virtual ICollection<StatusesWorker> StatusesWorkers { get; set; } = new List<StatusesWorker>();

    public virtual ICollection<StatusesWorkspace> StatusesWorkspaces { get; set; } = new List<StatusesWorkspace>();
}

File: frontend/src/app/services/models/Worker.ts
import { IStatusesWorker } from "./StatusesWorker";
import { IStatusesWorkspace } from "./StatusesWorkspace";

export interface IWorker {
    idWorker: number;
    name: string;
    surname: string;
    patronymic?: string;
    isDeleted: boolean;
    statusesWorkers?: IStatusesWorker[];
    statusesWorkspaces?: IStatusesWorkspace[];
}